egs/wsj/s5/steps/nnet3/decode.sh

```diff
@@ -68,7 +68,7 @@ model=$srcdir/$iter.mdl
 extra_files=
 if [ ! -z "$online_ivector_dir" ]; then
   steps/nnet2/check_ivectors_compatible.sh $srcdir $online_ivector_dir || exit 1
-  extra_files="$srcdir/final.ie.id $online_ivector_dir/ivector_online.scp $online_ivector_dir/ivector_period"
+  extra_files="$online_ivector_dir/ivector_online.scp $online_ivector_dir/ivector_period"
```

Comment:
OK, seems either someone has already fixed it or I;m not sure what's going on. But the decode.sh in master certainly does not have that issue.